### PR TITLE
upgrade K8s to 1.14

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/controller/v7/cloudconfig/master_template.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 
 	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/certs"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_4_1_2"
```
Comment (MarcelMue) [inline service/controller/v7/cloudconfig/master_template.go]: We also seem to have skipped `v_4_2_0` here.

File: Gopkg.toml [hunk 2]
```diff
@@ -75,7 +75,7 @@ required = [
   name = "github.com/giantswarm/exporterkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "add-1-14"
   name = "github.com/giantswarm/k8scloudconfig"
```
Comment (xh3b4sd) [inline Gopkg.toml]: See https://github.com/giantswarm/k8scloudconfig/pull/524.

File: service/controller/v7/cloudconfig/master_template.go (hunk 1 above)
Comment (xh3b4sd) [reply service/controller/v7/cloudconfig/master_template.go]: As we discussed in Azure we try to make the bump and see how it goes.

**Comment by mazzy89** [comment]
I've tested in ghost and the cluster works like a charm.

**Comment by mazzy89** [comment]
ready to be reviewed

**Comment by xh3b4sd** [comment]
Cannot approve. Is my PR. Looks good. Thank you.

File: service/controller/v7/version_bundle.go [hunk 3]
```diff
@@ -12,6 +12,16 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Pin calico-kube-controllers to master.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "kubernetes",
```
Comment (MarcelMue) [inline service/controller/v7/version_bundle.go]: Why was calico not added here? It should be on 3.6.1 now.

Comment (mazzy89) [reply service/controller/v7/version_bundle.go]: Good catch

**Pull request merged**